src/main/java/frc/robot/commands/Driver.java

```diff
@@ -4,23 +4,14 @@
 
 package frc.robot.commands;
 
-import frc.robot.subsystems.ExampleSubsystem;
 import edu.wpi.first.wpilibj2.command.CommandBase;
+import frc.robot.RobotContainer;
 
-/** An example command that uses an example subsystem. */
-public class ExampleCommand extends CommandBase {
-  @SuppressWarnings({"PMD.UnusedPrivateField", "PMD.SingularField"})
-  private final ExampleSubsystem m_subsystem;
-
-  /**
-   * Creates a new ExampleCommand.
-   *
-   * @param subsystem The subsystem used by this command.
-   */
-  public ExampleCommand(ExampleSubsystem subsystem) {
-    m_subsystem = subsystem;
+public class Driver extends CommandBase {
+  /** Creates a new drive. */
+  public Driver() {
     // Use addRequirements() here to declare subsystem dependencies.
-    addRequirements(subsystem);
+    addRequirements(RobotContainer.driveTrain);
```

Comment:
Use the `drivetrain` as parameter, do not access it directly.

Accessing it directly and excusing it for the field being static tangles the code and makes it harder to follow. One will read through `RobotContainer.java` and ask themselves why `drivetrain` is static with no answer until reaching this section.

```suggestion

```